src/sql/parts/grid/media/flexbox.css

```diff
@@ -3,10 +3,6 @@
     width: 100%;
 }
 
-.headersVisible > .fullsize {
```

Comment:
I think the height is platform specific and depends on if you are running a packaged build vs. the dev build.  Have you tested this with the packaged build for both Windows and macOS?

Reply:
Height looks the same on Windows and Mac on the dev build, but I'll check Linux and the packaged builds too

Reply:
thanks.  I've been checking a few configurations as well.  And everything looks good.  I want to check the mac packaged build and then I think that's probably enough validation to take into the release branch.

Reply:
...looks good on mac packaged builds too.  I'll merge to release and queue a `stable` release candidate build.